store/postgres/src/entities.rs

```diff
@@ -454,7 +455,19 @@ impl Connection {
         block: BlockNumber,
     ) -> Result<Vec<Entity>, QueryExecutionError> {
         match &*self.storage {
-            Storage::Json(json) => json.query(&self.conn, collection, filter, order, range),
+            Storage::Json(json) => {
+                // JSON storage can only query at the latest block
+                if block != BLOCK_NUMBER_MAX {
+                    return Err(StoreError::QueryExecutionError(
+                        "This subgraph uses JSONB storage, which does not \
+                         support querying at a specific block height. Redeploy \
+                         the subgraph to remove this error message."
```

Comment:
`remove this error` -> `to enable this feature` maybe?

Reply:
Also, simply redeploying won't do it if the deployment ID doesn't change. So perhaps it should say `Deploy a new version of this subgraph to enable this feature`?

Reply:
Yes on both counts. Changed.